src/ext/BalExtension/mba/core/IManagedBootstrapperEngine.cs

```diff
@@ -0,0 +1,44 @@
+﻿//-------------------------------------------------------------------------------------------------
+// <copyright file="IManagedBootstrapperEngine.cs" company="Outercurve Foundation">
+//   Copyright (c) 2004, Outercurve Foundation.
+//   This software is released under Microsoft Reciprocal License (MS-RL).
+//   The license and further copyright text can be found in the file
+//   LICENSE.TXT at the root directory of the distribution.
+// </copyright>
+//-------------------------------------------------------------------------------------------------
+
+namespace WixToolset.Bootstrapper
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Security;
+    using System.Text;
+
+    public interface IManagedBootstrapperEngine
+    {
+        IVariables<long> NumericVariables { get; }
```

Comment:
Should probably copy the XML comments from the class for IntelliSense documentation for anyone implementing the interface.